package.json

```diff
@@ -7,20 +7,32 @@
     "url": "react-bootstrap/react-bootstrap"
   },
   "homepage": "https://react-bootstrap.github.io/",
+  "main": "lib/index.js",
+  "jsnext:main": "es/index.js",
+  "module": "es/index.js",
   "scripts": {
     "bootstrap-docs": "npm run build lib && cd www && yarn install && cd ..",
     "build": "node tools/build.js",
     "build-docs": "npm run bootstrap-docs && npm -C www run build",
-    "lint": "eslint www/src src test tools *.js",
+    "ci-lint": "eslint --rule 'prettier/prettier: 2'",
+    "format": "npm run ci-lint -- --fix",
+    "lint": "npm run eslint .",
```

Comment:
I think this is a better pattern. I really don't like having the in-editor prettier lint errors, it's very nosy and something that is fixed for me, so its not helpful. This way it's only run in a CI or precommit setting.